src/behaviors/Behavior.js

```diff
@@ -1086,7 +1086,7 @@ var Behavior = Base.extend('Behavior', {
      * @return {boolean} Can re-order columns.
      */
     isColumnReorderable: function() {
-        return this.grid.properties.columnsReorderable;
+        return this.deprecated('isColumnReorderable()', 'grid.properties.columnsReorderable', '2.1.3');
```

Comment:
We got rid of fixed columns?

Reply:
does this have something to do with fixed columns? in any case, i'm just deprecated the unnecessary method in favor of the raw property. Recall that `deprecated()` just warns while still providing the original functionality.